src/_locales/en/messages.json

```diff
@@ -115,5 +115,24 @@
     },
     "partialMatchFullPath": {
         "message": "Search for keywords within the entire folder path"
+    },
+    "operation_action_move": {
```

Comment:
Can you make these strings camelCase for consistency with the others?

Reply:
Done with the dot, as agreed.